sdk/cognitivelanguage/ai-language-text/src/util.ts

```diff
@@ -163,3 +163,24 @@ export function getOperationOptions<OptionsT extends TextAnalysisOperationOption
     rest,
   };
 }
+
+/**
+ *
+ * @param error error with the target in the JSON error pointer format "#/items/0
+ * @returns number: the position of the task with error
+ */
+export function extractErrorPointerIndex(error: ErrorModel): number {
+  if (!error.target) {
+    throw new Error("Error target is not defined to parse the error pointer");
```

Comment:
Should we consider just logging the error instead of throwing it? I mainly wonder if the customer care about this scenario? If the target field doesn't exist for some reason, could we just create an error ourselves in the failed actions with no associated errors with a generic error?

Reply:
I'm following Python implementation to raise an error when the target is not properly formatted [here](https://github.com/kristapratico/azure-sdk-for-python/blob/20bf5d10aa7671d71c54504a5f1d058d27354227/sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_response_handlers.py#L403) or when there is no error. Logging would make sense too. I'm curious what error code can we use to create a generic error? Would it be "NotFound" or "ServiceUnavailable"?

Reply:
I see, I don't think it makes sense to create an error ourselves then because we don't know how to. The error code is an important part of a REST error and in this case, we don't know it.

I am fine with throwing then.